app/labor/badge_rewarder.rb

```diff
@@ -49,14 +55,14 @@ def self.award_tag_badges
       winning_article = Article.where("score > 100").
         published.
         where.not(user_id: past_winner_user_ids).
-        order("score DESC").
+        order(score: :desc).
         where("published_at > ?", 7.5.days.ago). # More than seven days, to have some wiggle room.
         cached_tagged_with(tag).first
       if winning_article
         award_badges(
           [winning_article.user.username],
           tag.badge.slug,
-          "Congratulations on posting the most beloved [##{tag.name}](#{ApplicationConfig['APP_PROTOCOL'] + ApplicationConfig['APP_DOMAIN']}/t/#{tag.name}) post from the past seven days! Your winning post was [#{winning_article.title}](#{ApplicationConfig['APP_PROTOCOL'] + ApplicationConfig['APP_DOMAIN'] + winning_article.path}). (You can only win once per badge-eligible tag)",
+          "Congratulations on posting the most beloved [##{tag.name}](#{URL.tag(tag)}) post from the past seven days! Your winning post was [#{winning_article.title}](#{URL.article(winning_article)}). (You can only win once per badge-eligible tag)",
```

Comment:
Use URL helper.

Reply:
Should we not break this long single line string into multiple lines?

Reply:
We can, but it's in the middle of an argument list and I decided to rather leave it alone for now. No particular reason, I just figured we may soon need to find a better way to deal with copy anyway when dealing with I18n.